src/zenml/image_builders/build_context.py

```diff
@@ -14,18 +14,18 @@
 """Image build context."""
 
 import os
-from pathlib import Path
-from typing import IO, Dict, List, Optional, Set, Tuple, cast
+from typing import IO, Dict, List, Optional, Set, cast
 
 from zenml.constants import REPOSITORY_DIRECTORY_NAME
 from zenml.io import fileio
 from zenml.logger import get_logger
 from zenml.utils import io_utils, string_utils
+from zenml.utils.archivable import Archivable
 
 logger = get_logger(__name__)
 
 
-class BuildContext:
+class BuildContext(Archivable):
     """Image build context.
 
     This class is responsible for creating an archive of the files needed to
```

Comment:
Since you added a base class here now, the `__init__` is missing a `super().__init__()`.

Reply:
Yep agreed. If it's okay with you, I'll add that in my PR that is based on this, to avoid re-running the CI for this PR.

Reply:
Sure, go ahead!